presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java

```diff
@@ -513,12 +513,12 @@ private QueryResults createQueryResults(long token, UriInfo uriInfo, DispatchInf
         }
 
         /**
-         * Gets the URI where information about query can be found inside server if it is not completed.
+         * Gets the URI where information about query can be found inside the server if it is not completed.
```

Comment:
The word "Information" seems too generic. What is this information? The query metadata? The query result?
Let's try not to be so generic if possible.

Reply:
Changed to
```
Gets the URI where a query's process result can be found inside the server if it is not completed.
```